apps/api/src/app/events/e2e/bridge-trigger.e2e.ts

```diff
@@ -136,7 +137,7 @@ contexts.forEach((context: Context) => {
         }
       }
 
-      await triggerEvent(session, workflowId, subscriber, { name: 'test_name' }, bridge);
+      await triggerEvent(session, workflowId, subscriber.subscriberId, { name: 'test_name' }, bridge);
```

Comment:
There is some strange behaviour with the way the test harness for Subscribers works with the Subscriber JWT token that I couldn't get to the bottom of, but it's not particularly important to fix. So we use the `subscriber.subscriberId` here, and when testing execution for subscribers with preferences below, we instead use `subscriber._id`.